src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -57,7 +57,7 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
         ))
     } map { response =>
       response.status match {
-        case StatusCodes.OK => ()
+        case StatusCodes.Created => ()
```

Comment:
I made a comment on your other PR, but this is another reason why this kind of error handling sucks.